src/logic/operate.js

```diff
@@ -0,0 +1,19 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const [one, two] = [Big(numberOne), Big(numberTwo)];
+  switch (operation) {
+    case '+':
+      return one.plus(two).toString();
+    case '-':
+      return one.minus(two).toString();
+    case 'X':
+      return one.times(two).toString();
+    case '÷':
+      return one.div(two).toString();
```

Comment:
manage division by error.........You can display an error on screen or something (`'INFNITY`)....just handle it
![Screen Shot 2020-04-11 at 4 41 34 PM](https://user-images.githubusercontent.com/40712459/79048144-e4e9bb00-7c12-11ea-93cd-c1ff5042aace.png)